grafana/har-to-k6
# [chore] Update node to fix docker image vulnerabilities

This PR aims to fix vulnerabilities inherited by `node:16-slim` image.

![image](https://github.com/grafana/har-to-k6/assets/1382208/5b28bf5f-855e-42ff-a586-e4b32c1a9bce)


- Node `v16 => v18`
- CLI/Docker
  - Added `-s, --stdout` so that docker image can be used without mounting a volume
  - entrypoint defaults to be using `--stdout`
- Docs
  - Updated `README.md` to include docker usage


**How to test**
- Build docker image and run it as shown in `README.md`
  - Build: `docker build --tag har-to-k6:local .`
  - Run: `docker run har-to-k6:local ./example/inconsistent-content-type.har > loadtest.js`
    - Expected result is a `loadtest.js` file on the host system
  - Run: `docker run har-to-k6:local -o loadtest.js ./example/inconsistent-content-type.har`
    - Expected result is a `loadtest.js` file inside the container (old default).
  - Note: warnings and errors should not end up in the test script
- Run `./bin/har-to-k6` with `-stdout` and `--output` in different combinations to make sure that we get a desired result
